spec/support/active_record_classes.rb

```diff
@@ -2,27 +2,11 @@
 Dir["#{File.dirname(__FILE__)}/models/*.rb"].sort.each { |file| require file }
 
 ar_schema.instance_exec do
-  create_table :nullable_ids
   create_table :mongo_documents do |t|
     t.string :name
   end
 end
 
-class NullableId < ActiveRecord::Base
```

Comment:
I could not find any place where this class was being used. Let me know if I should restore it or try to create new tests with records that return nil on their `primary_key` option.